frontend/app/controllers/modals/edit-anime.js

```diff
@@ -3,7 +3,7 @@ import ModalMixin from '../../mixins/modals/controller';
 import VersionableMixin from '../../mixins/modals/versionable';
 /* global moment */
 
-export default Ember.ObjectController.extend(ModalMixin, VersionableMixin, {
+export default Ember.Controller.extend(ModalMixin, VersionableMixin, {
   startingDate: Ember.computed('startedAiring', function(k, v) {
```

Comment:
Shouldn't this be `model.startedAiring`? Same with down on 11 and 17 and 21, where it should probably be `model.finishedAiring`?